<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>guiBladeDesignerClass</class>
 <widget class="QMainWindow" name="guiBladeDesignerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1214</width>
    <height>794</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>guiBladeDesigner</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="plotLayout">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="Line" name="line_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>2</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>5</height>
         </size>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCustomPlot" name="globalPlot" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>15</verstretch>
         </sizepolicy>
        </property>
        <property name="baseSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>2</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>5</height>
         </size>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCustomPlot" name="additionalPlot" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>6</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>2</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>5</height>
         </size>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="toolsLayout">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="leftMargin">
       <number>20</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>20</number>
      </property>
      <property name="bottomMargin">
       <number>50</number>
      </property>
      <item>
       <widget class="QTabWidget" name="tab_panel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>480</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>480</height>
         </size>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab_parametrs">
         <attribute name="title">
          <string>Blade</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <widget class="QTableWidget" name="parametrsTable">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>270</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>480</height>
             </size>
            </property>
            <property name="showGrid">
             <bool>true</bool>
            </property>
            <property name="gridStyle">
             <enum>Qt::SolidLine</enum>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
            <property name="cornerButtonEnabled">
             <bool>true</bool>
            </property>
            <row>
             <property name="text">
              <string>xMax</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>yMax</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>alpha1</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>alpha2</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>R1</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>R2</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>omega1</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>omega2</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>RMax</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>x RMax</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>RBend</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>x RBend</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>alphaBend</string>
             </property>
            </row>
            <column>
             <property name="text">
              <string>value</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>increment</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_cascade">
         <attribute name="title">
          <string>Cascade</string>
         </attribute>
        </widget>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Maximum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="computeCamberButton">
          <property name="text">
           <string>compute Camberline </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="computeSuctionSideButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>compute Suction Side</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1214</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionRead"/>
    <addaction name="actionSave"/>
    <addaction name="actionExport_to_AutoCad"/>
    <addaction name="separator"/>
    <addaction name="actionConfig"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionRead"/>
   <addaction name="actionSave"/>
   <addaction name="actionExport_to_AutoCad"/>
   <addaction name="actionConfig"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionRead">
   <property name="icon">
    <iconset resource="guiBladeDesigner.qrc">
     <normaloff>:/guiBladeDesigner/Resources/Icons/if_ic_find_in_page_48px_352363.png</normaloff>:/guiBladeDesigner/Resources/Icons/if_ic_find_in_page_48px_352363.png</iconset>
   </property>
   <property name="text">
    <string>Read...</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:8pt; font-style:italic;&quot;&gt;Open and read parametrs from file&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="guiBladeDesigner.qrc">
     <normaloff>:/guiBladeDesigner/Resources/Icons/if_ic_save_48px_352084.png</normaloff>:/guiBladeDesigner/Resources/Icons/if_ic_save_48px_352084.png</iconset>
   </property>
   <property name="text">
    <string>Save as...</string>
   </property>
   <property name="toolTip">
    <string>Save as</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionConfig">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="guiBladeDesigner.qrc">
     <normaloff>:/guiBladeDesigner/Resources/Icons/if_ic_settings_48px_352095.png</normaloff>:/guiBladeDesigner/Resources/Icons/if_ic_settings_48px_352095.png</iconset>
   </property>
   <property name="text">
    <string>Config</string>
   </property>
  </action>
  <action name="actionExport_to_AutoCad">
   <property name="icon">
    <iconset resource="guiBladeDesigner.qrc">
     <normaloff>:/guiBladeDesigner/Resources/Icons/if_export-outline_216189.png</normaloff>:/guiBladeDesigner/Resources/Icons/if_export-outline_216189.png</iconset>
   </property>
   <property name="text">
    <string>Export to AutoCad</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="guiBladeDesigner.qrc"/>
 </resources>
 <connections/>
</ui>
